src/main/scala/viper/gobra/reporting/VerifierError.scala

```diff
@@ -263,6 +286,11 @@ case class InsufficientPermissionError(info: Source.Verifier.Info) extends Verif
   override def message: String = s"permission to ${info.origin.tag.trim} might not suffice"
 }
 
+case class InsufficientPermissionError2(tag: String) extends VerificationErrorReason {
```

Comment:
Would probably rename it to something else such as `InsufficientPermissionToTagError`